cdc/owner/changefeed.go

```diff
@@ -87,6 +87,8 @@ type changefeed struct {
 	initialized bool
 	// isRemoved is true if the changefeed is removed
 	isRemoved bool
+	// isReleased is true if the changefeed's resource is released
+	isReleased bool
```

Comment:
`initialized` / `isRemoved` / `isReleased`.

Do we really need so many variables?

Reply:
or we can add a status field for changefeed